lib/st_package_reviewer/__main__.py

```diff
@@ -0,0 +1,236 @@
+import argparse
+import io
+import logging
+from pathlib import Path
+import re
+import sys
+import tempfile
+import textwrap
+
+from github3 import GitHub
+
+from . import set_debug, debug_active, __version__
```

Comment:
Importing some non-existent stuff. Seems unfinished/untested.

Reply:
Note that you might be reviewing the st_package_reviewer dependency instead of the action that wraps it. And that dependency is currently a mishmash of versions of the package reviewer script...